spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'helpers/comments_helper_spec.rb'
+
+
+RSpec.describe Api::V1::CommentsController, type: :controller do
+  before(:each) do
+    @user = build(:user)
+    @user.confirm
+    sign_in @user
+    @post = create(:post)
+  end
+
+  describe 'CREATE POST / ' do
+    context 'successful creation' do
+      it 'responds with 201' do
+        create_comment('Hello, potato!', @post.id)
+        expect(response).to have_http_status(201)
+      end
+
+      it 'returns new comment json' do
+        create_comment('Hello json', @post.id)
+        json = JSON.parse(response.body)
+        expect(json["id"]).to eq(1)
+        expect(json["body"]).to eq("Hello json")
+        expect(json["user_id"]).to eq(1)
+        expect(json["post_id"]).to eq(1)
+      end
+
+      it 'creates a comment' do
+        create_comment('Hello, world!', @post.id)
+        expect(Comment.find_by(body: 'Hello, world!')).to be
+      end
+    end
+
+    context 'unsuccessful creation...' do 
+    end
+  end
+
+  describe 'DELETE Comments' do
+    it 'deletes a comment' do
+      create_comment('Hello, potato!', @post.id)
```

Comment:
You could DRY out some of these tests by moving repeated code into before(:each) blocks.